### PR TITLE
Use "delphix/actions" for shfmt

### PR DESCRIPTION
**Pull request by prakashsurya**

### CODE REVIEW

**Comment by prakashsurya** [comment]
This is depends on: https://github.com/delphix/actions/pull/3 and https://github.com/delphix/actions/pull/4

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -118,5 +118,4 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
-      - run: ./.github/scripts/install-shfmt.sh
-      - run: /usr/local/bin/shfmt -d .
+      - uses: delphix/actions/shfmt@master
```
Comment (pzakha) [inline .github/workflows/main.yml:121]: We don't also run shellcheck for sdb?

Comment (prakashsurya) [reply .github/workflows/main.yml:121]: I don't see it used anywhere in this repository, do you? Did I overlook something?

Comment (pzakha) [reply .github/workflows/main.yml:121]: Sorry, I meant that as a generic question. @sdimitro Is there an explicit reason we aren't running shellcheck? Should we add it?

Comment (sdimitro) [reply .github/workflows/main.yml:121]: No explicit reason - I just only knew about `shfmt` at the time when I made that change :-)

Comment (pzakha) [reply .github/workflows/main.yml:121]: Okay, we can perhaps consider adding it in a later commit.

**Pull request merged**